include/RAJA/RAJA.hxx

```diff
@@ -63,39 +63,60 @@
 
 #include "RAJA/config.hxx"
 
-#include "RAJA/internal/defines.hxx"
+#include "RAJA/util/Defines.hxx"
 
-#include "RAJA/Types.hxx"
+#include "RAJA/util/Types.hxx"
 
-#include "RAJA/operators.hxx"
-#include "RAJA/reducers.hxx"
+#include "RAJA/util/Operators.hxx"
+
+//
+// All platforms must support sequential execution.
+//
+#include "RAJA/policy/Sequential.hxx"
+
+//
+// All platforms should support simd execution.
+//
+#include "RAJA/policy/Simd.hxx"
+
+#if defined(RAJA_ENABLE_CUDA)
+#include "RAJA/policy/Cuda.hxx"
+#endif
+
+#if defined(RAJA_ENABLE_OPENMP)
+#include "RAJA/policy/OpenMP.hxx"
+#endif
+
+#if defined(RAJA_ENABLE_CILK)
+#include "RAJA/policy/Cilk.hxx"
+#endif
 
 //
 // Strongly typed index class.
 //
-#include "RAJA/IndexValue.hxx"
+#include "RAJA/index/IndexValue.hxx"
 
 //
 // Generic iteration templates require specializations defined
 // in the files included below.
 //
-#include "RAJA/forall.hxx"
+#include "RAJA/pattern/Forall.hxx"
```

Comment:
As in my earlier comment, I think the form of this file name and the policy stuff above should match the style of the forall methods and policies, etc.